samples/CMakeLists.txt

```diff
@@ -151,7 +151,7 @@ endfunction()
 add_mesh("assets/models/monkey/monkey.obj" "suzanne.filamesh")
 
 set (COMPRESSION "--compression=uastc")
-add_ktxfiles("assets/models/monkey/albedo.png" "albedo.ktx2" "${COMPRESSION}")
```

Comment:
prideout!

Reply:
This change means we need to change loading code somewhere?

Reply:
well that's why I kept the `albedo.ktx2`

Reply:
lol ah yes it is.